bin/host/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "host"
+version = "0.1.0"
+edition.workspace = true
+license.workspace = true
+authors.workspace = true
+repository.workspace = true
+homepage.workspace = true
+exclude.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+# workspace
+anyhow.workspace = true
+tracing.workspace = true
+alloy-primitives.workspace = true
```

Comment:
Activate the `serde` feature so that `B256` and other primitives from alloy implement `serde::Serialize`

Reply:
Yeah this removes the need for your custom value parser

Reply:
Yeah for sure - given that this is a bin crate, think we can just always enable serde. Folks shouldn't be consuming this code downstream.